### DISCUSSION TITLE
Preserving directory structure?

### DISCUSSION THREAD

**Discussion by DavSanchez** [pyload/pyload#3983] [Help]
Hi guys!

### Question

I was wondering if there is a way to download files while preserving the same directory structure as the source contents? For example, if a mega link contains three folders with different files in each one, the download would mimic this in the destination by creating the three directories and placing the files in each one as in the source.

From what I've tried, `pyLoad` just downloads everything in the same place, and but maybe I am missing something. If it's not possible currently, would this be a feasible feature request?

Thanks!

**Comment by GammaC0de** [comment]
> If it's not possible currently,

Yes, currently it's not possible.

> would this be a feasible feature request?

Yes but we are very short on free time
